keyboards/maxr1998/phoebe/phoebe.c

```diff
@@ -0,0 +1,5 @@
+#include "phoebe.h"
+
+void matrix_init_kb(void) {
```

Comment:
This can be safely removed, since it's not being used.

Reply:
Hope it's ok now.